src/ja/common/proxy/ssh.py

```diff
@@ -161,6 +166,8 @@ def to_dict(self) -> Dict[str, object]:
 
     @classmethod
     def from_dict(cls, property_dict: Dict[str, object]) -> "SSHConfig":
+        if property_dict is None:
```

Comment:
There were multiple classes where a property of type SSHConnection could be None.
I figured that instead of checking externally if the dict is None this would be simpler.